app/MainHie.hs

```diff
@@ -136,12 +143,20 @@ run opts = do
       Just err -> error (pdeErrorMsg err)
       Nothing -> return ()
 
+    when (optDumpSwagger opts) $ do
+      -- putStrLn "dumping swagger definition"
+      let swagger = hieSwagger plugins
+      -- let swagger = hieSwagger2 (recProxy taggedPlugins)
+      -- putStrLn (show swagger)
+      putStrLn (B.unpack $  encode swagger)
```

Comment:
Is this guaranteed to be ascii? Otherwise Char8 is evil and this should be properly decoded (or just dump the bytestring).